### DISCUSSION TITLE
First request made after starting up the dev server takes too much time

### DISCUSSION THREAD

**Discussion by aiupjulian** [vercel/next.js#68806] [Help]
### Summary

Hello,
I have an issue in a next js project where the first request to the dev server takes a lot of time to complete (around 1 minute). I ran the profiler and noticed that most of that time is because `FSWatcher.close` is running. The rest of the requests take between 5 and 10 secs to complete, and I don't see these calls to `FSWatcher.close` anymore. Is this normal?
Bear in mind that I'm working on a monorepo and each page compiles around 4k modules.

First page load:
<img width="1919" alt="first-page-load" src="https://github.com/user-attachments/assets/1138dd9e-151e-4967-82c0-37023560b58f">

Second page load:
<img width="1920" alt="second-page-load" src="https://github.com/user-attachments/assets/5ad9da7a-1017-4fa1-8aeb-3ceb37b43c24">

Thanks in advance!

### Additional information

```js
Operating System:
  Platform: darwin
  Arch: arm64
  Version: Darwin Kernel Version 23.5.0: Wed May  1 20:19:05 PDT 2024; root:xnu-10063.121.3~5/RELEASE_ARM64_T8112
  Available memory (MB): 16384
  Available CPU cores: 8
Binaries:
  Node: 20.16.0
  npm: 10.8.1
  Yarn: N/A
  pnpm: 8.6.12
Relevant Packages:
  next: 14.2.5 // Latest available version is detected (14.2.5).
  eslint-config-next: 13.4.19
  react: 18.2.0
  react-dom: 18.2.0
  typescript: 5.1.6
Next.js Config:
  output: standalone
```

**Comment by wwwdepot** [comment]
like starting up a car engine, you gotta wait for it to warm up 🤣
